yarn.lock

```diff
@@ -0,0 +1,5230 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Does this have to be commited?

Reply:
According to the yarn documentation: yes, to ensure everyone installs the exact same version of each dependency; But if everyone else just uses npm i can remove it

Reply:
I personally prefer npm since I am not familiar with yarn.